.python-version

```diff
@@ -0,0 +1,2 @@
+data-store
```

Comment:
I don't think you should be checking this in.  You should put this in your private gitignore (i.e., `.git/info/exclude`).

Reply:
Agreed. My mistake!

Reply:
My bad. Thanks for the tip re: private gitignore!